modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+.Procedure
+
+. Choose the watchdog action. This defines how the watchdog proceeds when the operating system (OS) no longer responds.
+
++
+.Available actions
+[horizontal]
+`poweroff`:: The virtual machine (VM) powers down immediately.
+`reset`:: The VM reboots in place, and the guest OS cannot react. Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, you should avoid their use.
+`shutdown`:: The VM gracefully powers down by stopping all services.
+
+. Create a YAML file and apply it to the cluster.
++
+The example below configures the `i6300esb` watchdog device on a RHEL8 VM with the poweroff action and exposes the device as `/dev/watchdog`.
++
+[source,yaml]
+----
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm2-rhel84-watchdog
+  name: vm2-rhel84-watchdog
+spec:
+  running: false
+  template:
+    metadata:
+     labels:
+        kubevirt.io/vm: vm2-rhel84-watchdog
+    spec:
+      domain:
+        devices:
+          watchdog:
+            name: mywatchdog
+            i6300esb:
+              action: "poweroff"
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: "1Gi"
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - containerDisk:
+          image: registry.redhat.io/rhel8/rhel-guest-image
+        name: containerdisk
+      - cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            password: redhat
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+----
+
+This device can now be used by the watchdog binary.
+
+.Verification
+Confirm watchdog functionality on non-production systems.
+
+[WARNING]
+====
+This procedure is provided for testing the functionality of watchdogs only and must not be run on production machines.
+====
+
+. Run the following command on the virtual machine to confirm the VM identified the watchdog card:
+
++
+[source,terminal]
+----
+$ lspci | grep watchdog -i
+----
+
+. Run one of the following commands to confirm the watchdog is active:
```

Comment:
I'm not sure I would even make this into a choice. I would just make the kernel panic and the process kill into separate steps. 

But if you want to keep it as a choice, make the two commands into bullet list items.